AIPs/aip-5.md

```diff
@@ -1,212 +1,178 @@
 
-To create multiplicative symmetry let's begin by observing that:
-* <code>For every scaling factor S, there exists an inverse scaling factor S<sup>-1</sup> such that S * S<sup>-1</sup> = 1</code>
+The proposed values are:
 
-And let’s also observe that:
 
-* <code>For every price P there exists an inverse price P<sup>-1</sup> such that P * P<sup>-1</sup> = 1</code>
+```
+l (Lower) = -0.11
+u (Upper) = 0.11 
+g (Growth) = 4 
+```
 
-We can enforce the constraint of “balanced” supply-change-factors by computing contraction supply-change-factors as the inverse of expansion supply-change-factors. In other words: 
 
 <p align="center">
-<img src="../assets/aip-5/piecewise_eq.png" alt="drawing" width="380"/>
+<img src="../assets/aip-5/sigmoid.png" alt="sigmoid"/>
 </p>
 
-This way, for every price pair <code>{P, P<sup>-1</sup>}</code> the corresponding supply-change-factor pair <code>{S, S<sup>-1</sup>}</code> upholds the constraint that  <code>S * S<sup>-1</sup> = 1</code>.
+
+```
+Here we can see the upper asymptote approaches a scaling factor of 1.11 at a normalized price of $3.
+```
+
+
+To show that the proposed (sigmoid) expansion output is reasonably similar to the current (linear) output:
+
+1. Proposed and current supply policy functions were applied to historic VWAP data.
+2. Aggregate positive rebases were computed for proposed and current supply functions.
+3. The ratio between proposed and current aggregate positive rebases is: 1.055
+
+Based on our analysis above, the function parameters above meet the goal of having a similar aggregate effect on expansion rebases. 
+
+**_Normalized VWAP distribution_**
+
 
 <p align="center">
-<img src="../assets/aip-5/balanced_sigmoid_latex_thin.jpg" alt="drawing" width="700"/>
+<img src="../assets/aip-5/price_distribution.png" alt="normalized price distribution"/>
 </p>
 
-**2.2. _Balanced Sigmoid Equation and Parameters_**
+**_Proposed vs Current Rebase Totals_**
 
-Recall that the sigmoid supply change function accepts a price deviation and returns a supply change percentage. Combining it with the inversing function gives the output<sup>*</sup>: 
 
 <p align="center">
-<img src="../assets/aip-5/piecewise_sigmoid_eq.png" alt="drawing" width="380"/>
+<img src="../assets/aip-5/rebase_histogram.png" alt="rebase histogram"/>
 </p>
 
-*_<sup>\* In the equation above, P denotes normalized price, P = vwap_price/target</sup>_*
+### Test Cases
+
+Existing unit tests will be updated with the correct calculation results to maintain test coverage.
+
+
+### Conclusions
+
+This change is expected to make AMPL more robust as a base money in decentralized finance applications. AMPL will keep its dynamic properties to change supply to match demand. In addition AMPL will be enhanced by protections against short-lived but extreme market conditions. The protocol is also expected to remain closer to its target, now that larger supply adjustments will happen closer to the target price.
```

Comment:
```suggestion
This change is expected to make AMPL more robust as a base money in decentralized finance applications. The Ampleforth protocol's fundamental nature has NOT been altered, it will keep its dynamic properties to change supply to match demand. In addition, AMPL will be enhanced by protections against short-lived but extreme market conditions. The protocol is also expected to remain closer to its target, now that larger supply adjustments will happen closer to the target price.
```

Reply:
Edited based on your feedback here and Brandon's feedback.